superset-frontend/src/components/Menu/Menu.tsx

```diff
@@ -134,6 +134,19 @@ const StyledHeader = styled.header`
     display: flex;
     align-items: center;
   }
+
+  .ant-menu {
+    .ant-menu-item-group-title {
+      padding-bottom: ${({ theme }) => theme.gridUnit}px;
+    }
+    .ant-menu-item {
+      margin-bottom: ${({ theme }) => theme.gridUnit * 2}px;
+    }
+    .about-section {
+      margin: ${({ theme }) => theme.gridUnit}px 0
+        ${({ theme }) => theme.gridUnit * 2}px;
+    }
```

Comment:
Can we use `:last-child` so this can be more generic?

Reply:
Those styles are needed not for any last child but specifically for "About us" section. That section is a bit "hacked" as "Version" and "SHA"  are not defined as menu items - we don't want them to be clickable or highlighted on hover. Because of that, default Antd styles from `Menu` were not applied there and I added those styles to make it look nicer.

Reply:
In that case, we may still generalize it something like `MetaMenuItem` in case this pattern is used elsewhere. But let's not over-optimize for now.